app/src/main/contrib/actions/EstimateActions.ts

```diff
@@ -1,14 +1,21 @@
-import { FetchActions, FetchActionsInterface } from "../../shared/actions/FetchActions";
-import { alt } from "../../shared/alt";
+import {FetchActions, FetchActionsInterface} from "../../shared/actions/FetchActions";
+import {alt} from "../../shared/alt";
 
 interface Actions extends FetchActionsInterface<string> {
     clearUsedToken(): boolean;
+
+    setRedirectPath(path: string): string;
```

Comment:
This has to be separated into it's own action as is called at the page level, whereas the token needs to be fetched by the form, once the burden estimate set has been created and we have an id